src/login/logind-action.c

```diff
@@ -113,11 +113,12 @@ static const HandleActionData action_table[_HANDLE_ACTION_MAX] = {
         },
 };
 
-const HandleActionData* manager_item_for_handle(HandleAction handle) {
-        assert(handle >= 0);
-        assert(handle < (ssize_t) ELEMENTSOF(action_table));
+const HandleActionData* handle_action_lookup(HandleAction action) {
 
-        return &action_table[handle];
+        if (action < 0 || (size_t) action >= ELEMENTSOF(handle_action_data_table))
```

Comment:
This is more than renaming, it's a slight change in behaviour - which is fine, but please mention it in the commit message (commit: `logind: rename manager_item_for_handle() → handle_action_lookup()`)